docs/content/users/quickstart.md

```diff
@@ -1,274 +1,230 @@
+    cd ExpressionEngine
+    ddev config # Accept the defaults
+    ddev start
+    ddev composer install
+    touch system/user/config/config.php
+    echo "EE_INSTALL_MODE=TRUE" >.env.php
+    ddev start
+    ddev launch /admin.php  # Open installation wizard in browser
+    ```
 
-        ### Git Clone
+    When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
 
-        ```bash
-        git clone https://github.com/example/my-drupal-site
-        cd my-drupal-site
-        ddev config # Follow the prompts to set Drupal version and docroot
-        ddev composer install # If a composer build
-        ddev launch
-        ```
+## Laravel
 
-    === "Backdrop"
-
-        ### Backdrop
-
-        To get started with [Backdrop](https://backdropcms.org), clone the project repository and navigate to the project directory.
-
-        ```bash
-        git clone https://github.com/example/example-site
-        cd example-site
-        ddev config
-        ddev start
-        ddev launch
-        ```
-
-=== "ExpressionEngine"
-
-    ## ExpressionEngine
-
-    === "ExpressionEngine ZIP File Download"
-
-        Download the ExpressionEngine code from [expressionengine.com](https://expressionengine.com/), then follow these steps based on the [official installation instructions](https://docs.expressionengine.com/latest/installation/installation.html):
-
-        ```bash
-        mkdir my-ee && cd my-ee
-        unzip /path/to/ee-zipfile.zip
-        ddev config # Accept the defaults
-        ddev start
-        ddev launch /admin.php # Open installation wizard in browser
-        ```
-
-        When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
-
-        Visit your site.
-
-    === "ExpressionEngine Git Checkout"
+Use a new or existing Composer project, or clone a Git repository.
 
-        Follow these steps based on the [ExpressionEngine Git Repository README.md](https://github.com/ExpressionEngine/ExpressionEngine#how-to-install):
-
-        ```bash
-        git clone https://github.com/ExpressionEngine/ExpressionEngine # for example
-        cd ExpressionEngine
-        ddev config # Accept the defaults
-        ddev start
-        ddev composer install
-        touch system/user/config/config.php
-        echo "EE_INSTALL_MODE=TRUE" >.env.php
-        ddev start
-        ddev launch /admin.php  # Open installation wizard in browser
-        ```
-
-        When the installation wizard prompts for database settings, enter `db` for the _DB Server Address_, _DB Name_, _DB Username_, and _DB Password_.
-
-=== "Laravel"
-
-    ## Laravel
+The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) just as it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```

Comment:
```suggestion
The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) like it can for Laravel. DDEV automatically updates or creates the `.env` file with the database information.
```